### ISSUE TITLE
Build sometimes terminated after one of many test containers exited 0

### ISSUE THREAD

**Issue by solsson** [Yolean/build-contract#25]
I don't have a clear repro but I've seen this before and now I added a third test container to a docker-compose.yml that had two already. Once the third test exits, build-contract goes:
```
dockercompose_hooks-test_1 exited with code 0
Killing configstoredockercompose_site-admin-test_1 ...
```
where both hooks-test and site-admin-test have:
```
    labels:
      - com.yolean.build-contract
```

**Comment by atamon** [comment]
Might be related to https://github.com/Yolean/build-contract/issues/24 ?

**Comment by solsson** [comment]
Tried with `docker-compose -f build-contracts/docker-compose.yml down --remove-orphans` but still getting unwarranted killings. Did we have any other workarounds to try?

**Comment by atamon** [comment]
If you're okay with removing all of your docker containres: docker rm -f
$(docker ps -aq)

Otherwise no :-/

**Comment by solsson** [comment]
Did both `docker system prune` and verified that there is no output from `docker ps -aq -f label=com.yolean.build-contract`, i.e. no build-contract remnants from any project.

**Comment by solsson** [comment]
I think #24 was the issue, because after the cleanup above I finally noticed that I had introduced a test failure. Not easy to see when your tests get killed randomly in the middle, and the build-contract outcome was the same as before cleanup :)

The failure taught me a bash-for-testing gotcha:
```
./my-brilliant-script some args
[[ $? -eq 0 ]] && echo "Expected that command to fail" && exit 1

exit 0
```
The assertion worked, but without the `exit 0` in the end the script would exit 1 anyway. I guess the above can be done more elegantly.
